.github/workflows/publish.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: cargo build --release --target wasm32-unknown-unknown --examples
 
       - name: Install wasm-bindgen-cli
-        run: cargo install --force wasm-bindgen-cli --version 0.2.73
+        run: cargo install wasm-bindgen-cli
```

Comment:
How do we know that `wasm-bindgen-cli` version matches the one used during `cargo build` above?

Reply:
Since wgpu-rs doesn't have a lock file, shouldn't both pick up the latest one? (apart from differences in major version)
Maybe there's a better way of doing this that I don't know

Reply:
Yeah generally I think that's true. It's possible they might not match in some cases (e.g. if a dependency or transitive dependency requires a particular version or version range) although those cases are probably unlikely, and if/when that happens we can pin CI to a particular version or workaround it. So we can probably proceed with this